zeebe/engine/src/test/java/io/camunda/zeebe/engine/state/deployment/RpaStateTest.java

```diff
@@ -0,0 +1,68 @@
+/*
+ * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH under
+ * one or more contributor license agreements. See the NOTICE file distributed
+ * with this work for additional information regarding copyright ownership.
+ * Licensed under the Camunda License 1.0. You may not use this file
+ * except in compliance with the Camunda License 1.0.
+ */
+package io.camunda.zeebe.engine.state.deployment;
+
+import static io.camunda.zeebe.util.buffer.BufferUtil.wrapString;
+import static org.assertj.core.api.Assertions.assertThat;
+
+import io.camunda.zeebe.engine.state.mutable.MutableProcessingState;
+import io.camunda.zeebe.engine.state.mutable.MutableRpaState;
+import io.camunda.zeebe.engine.util.ProcessingStateExtension;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+
+@ExtendWith(ProcessingStateExtension.class)
+public class RpaStateTest {
```

Comment:
🔧 We're missing some additional tests about storing data in the column families. You can see how this is covered in the `DecisionStateTest` [here](https://github.com/camunda/camunda/blob/ee185e1d52eacec6802f07daf42341db9a0ff19e/zeebe/engine/src/test/java/io/camunda/zeebe/engine/state/deployment/DecisionStateTest.java#L103-L955).